frontend/test/metabase/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -99,6 +106,206 @@ const TEST_TABLE = "scoreboard_actions";
           expect(result.rows.length).to.equal(1);
         });
       });
+
+      it("adds an implicit create action to a dashboard and runs it", () => {
+        createModelFromTable(TEST_TABLE);
+        cy.get("@modelId").then(id => {
+          cy.request({
+            url: "/api/action",
+            method: "POST",
+            body: {
+              kind: "row/create",
+              name: "Create",
+              type: "implicit",
+              model_id: id,
+            },
+          });
```

Comment:
FWIW, you can use [`createAction` helper](https://github.com/metabase/metabase/blob/cf016b2965da75a98fe8fbc8418c96fee2faa1aa/frontend/test/__support__/e2e/helpers/e2e-action-helpers.js#L18). It's not much yet, but it'd make sense to extend it with some defaults eventually